src/interactors/interactors.test.js

```diff
@@ -70,4 +77,18 @@ describe('Interactors', () => {
     await updateSchema.execute(mockThing);
     expect(cloudMocks.mockUpdateSchema).toHaveBeenCalled();
   });
+
+  test("publishData: should execute interactor correctly when publish thing's data on cloud", async () => {
+    const cloud = new Cloud();
+    const publishData = new PublishData(cloud);
+    await publishData.execute(mockThing.id, mockData);
+    expect(cloudMocks.mockPublishData).toHaveBeenCalled();
+  });
+
+  test("publishData: should execute interactor without errors when fail to publish thing's data on cloud", async () => {
```

Comment:
Suggestion :point_down: 

```suggestion
  test("publishData: should execute interactor without errors when publishing a thing's data to cloud fails", async () => {
```

Reply:
Done!